pytraj/common_actions.py

```diff
@@ -737,15 +737,48 @@ def calc_multivector(traj=None,
 
 
 @_super_dispatch()
-def calc_volmap(traj=None, mask="", top=None, dtype='ndarray', *args, **kwd):
-    command = mask
+def calc_volmap(traj, mask='',
+                grid_spacing='0.0. 0.0. 0.0',
+                buffer=3.0,
+                centermask='*',
+                radscale=1.36,
+                peakcut=0.05,
+                top=None,
+                dtype='ndarray',
+                frame_indices=None):
+    '''(cpptraj doc) Grid data as a volumetric map, similar to the
+    ’volmap’ command in VMD. The density is calculated by treating each atom as a 
```

Comment:
you're right about copy and paste from PDF. I got this problem before with py2 and could not know why for a while.